addon/templates/components/frost-fixed-table.hbs

```diff
@@ -84,6 +84,7 @@
     }}
       {{#each items as |item index|}}
         {{frost-table-row
+          class=(if isSelectable 'selectable' '')
```

Comment:
so it is in frost-table-row, but in this case don't want to set isSelectable to true for middle and right sections otherwise they'll get their own selection cell too. Only want a selection cell in the left section